examples/embeddable_examples/public/react_embeddables/search/search_react_embeddable.tsx

```diff
@@ -134,13 +141,14 @@ export const getSearchEmbeddableFactory = (services: Services) => {
             );
           }
 
-          return (
-            <p>
-              Found <strong>{count}</strong> from {defaultDataView.name}
-            </p>
-          );
+          return dataLoading
+            ? <EuiLoadingSpinner size="l" />
```

Comment:
I'm not sure if we should show a spinner like this in our example. I think best practice for subsequent loads is to show the data you have (stale data) with an unobtrusive loader (the EUI indeterminate loader works well for this). This strategy minimizes layout shifting / flashing and allows the user to see the data change directly.

Right now Lens uses this strategy, but I'd like to codify this into the embeddable system soon. I don't think we should make an example of hiding the whole UI on every data load.

Reply:
ok, I have reverted changes to SearchEmbeddable and we can re-address at a later time